Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -225,6 +223,12 @@ inline WICPixelFormatGUID PixelFormat() const {
         _impl.renderingIntent = intent;
         return *this;
     }
+
+    ~__CGImage() {
+        for (auto listener: _imageDestructionListeners) {
+            listener(this);
+        }
```

Comment:
:o you read my mind

---
In reply to: [103062474](https://github.com/Microsoft/WinObjC/pull/2099#discussion_r103062474) [](ancestors = 103062474)